content/sensu-go/5.0/getting-started/sample-app.md

```diff
@@ -0,0 +1,363 @@
+
+{{< highlight shell >}}
+sensuctl configure
+? Sensu Backend URL: http://sensu.local
+? Username: demo
+? Password: ********
+? Namespace: demo
+? Preferred output format: tabular
+{{< /highlight >}}
+
+**7. Log in to the Sensu dashboard.**
+
+At http://webui.sensu.local/signin, sign in with your sensuctl username (`demo`) and password.
+
+## Deploying Sensu agents and InfluxDB
+
+**1. Deploy InfluxDB with a Sensu Agent sidecar**
+
+Create a Kubernetes ConfigMap for InfluxDB configuration
+
+{{< highlight shell >}}
+kubectl create configmap influxdb-config --from-file go/configmaps/influxdb.conf
+{{< /highlight >}}
+
+Deploy InfluxDB with a Sensu Agent sidecar
+
+{{< highlight shell >}}
+kubectl create -f go/deploy/influxdb.sensu.yaml
+{{< /highlight >}}
+
+**2. Create a Sensu pipeline to store metrics with InfluxDB.**
+
+Register a Sensu asset for the [Sensu InfluxDB handler][], and create an `influxdb` event handler.
+
+{{< highlight shell >}}
+sensuctl create -f go/config/assets/influxdb-handler.yaml
+
+sensuctl create -f go/config/handlers/influxdb.yaml
+{{< /highlight >}}
+
+**3. Deploy Sensu agent sidecars for the dummy app instances.**
+
+{{< highlight shell >}}
+kubectl apply -f go/deploy/dummy.sensu.yaml
+{{< /highlight >}}
+
+## Monitoring an app
+
+Let's take a look at what we're monitoring.
+
+{{< highlight shell >}}
+sensuctl entity list
+{{< /highlight >}}
+
+We can see the Sensu agents we have installed on the dummp app, InfluxDB, and Grafana pods with their last seen timestamp.
+
+{{< highlight shell >}}
+ID               Class    OS                   Subscriptions                           Last Seen            
+─────────────────────────── ─────── ─────── ─────────────────────────────────────────── ─────────────────────────────── 
+dummy-6c57b8f868-ft5dz      agent   linux   dummy,entity:dummy-6c57b8f868-ft5dz         2018-11-20 18:43:15 -0800 PST  
+dummy-6c57b8f868-m24hw      agent   linux   dummy,entity:dummy-6c57b8f868-m24hw         2018-11-20 18:43:15 -0800 PST  
+grafana-5b88f8df8d-vgjtm    agent   linux   grafana,entity:grafana-5b88f8df8d-vgjtm     2018-11-20 18:43:14 -0800 PST  
+influxdb-78d64bcfd9-8km56   agent   linux   influxdb,entity:influxdb-78d64bcfd9-8km56   2018-11-20 18:43:12 -0800 PST  
+{{< /highlight >}}
+
+### Create a Sensu pipeline to Slack
+Let’s say we want to receive a Slack alert if the dummy app returns an unhealthy response.
+We can create a Sensu pipeline to send alerts to a #demo channel in Slack using the [Sensu Slack plugin](https://github.com/sensu/sensu-slack-handler).
+Sensu Plugins are open-source collections of Sensu building blocks shared by the Sensu Community.
+
+**1. Create an asset to help agents find and install the [Sensu Slack handler](https://github.com/sensu/sensu-slack-handler)**
+
+{{< highlight shell >}}
+sensuctl create -f go/config/assets/slack-handler.yaml
+{{< /highlight >}}
+
+**2. Get your Slack webhook URL and add it to `go/config/handlers/slack.json`.**
```

Comment:
This particular file doesn't exist. I'm assuming this is to `go/config/handlers/slack.yaml`?